File: EjerciciosEntrega/MovPara.py
import numpy as np
import math
import matplotlib.pyplot as plt
v=float(input("Ingrese la velocidad inicial "))
a=float(input("Ingrese el ángulo "))
n=int(input("Ingrese la cantidad de intervalos "))
tiempo=[]
x=[0]
y=[0]
s=0
g=9.80665
rad=a*math.pi/180
vx=v*np.cos(rad)
vy=v*np.sin(rad)
t=2*vy/g
print("En el tiempo " ,s, "segundos, la velocidad en X es de " ,vx, " y en Y es de" ,vy,"")
for i in range(0, n):
    s=(t/n)+s;
    vy1=vy-(g*s);
    tiempo.append(s)
    print("En el tiempo " ,s, "segundos, la velocidad en X es de " ,vx, " y en Y es de" ,vy1,"")
for tem in tiempo:
    l=vx*tem
    f=vy*tem-0.5*g*tem**2
    x.append(l)
    y.append(f)
plt.plot(x,y,'o-')

File: EjerciciosEntrega/MovPara.cpp
#include <iostream>
#include <math.h>
#include<list>
#define PI 3.14159265
#define G 9.80665
using namespace std;

int main(){
    double a, v, vx, vy, t, rad, s, vy1;
    int n;
    cout << "Ingrese el valor de la velocidad inicial: ";
    cin >> v;
    cout << "Ingrese el valor del ángulo: ";
    cin >> a;
    cout << "Ingrese la cantidad de intervalos: ";
    cin >> n;
    s=0;
    rad=a*PI/180;
    vx=v*cos(rad);
    vy=v*sin(rad);
    t=2*vy/G;
    cout<< "El tiempo de vuelo es de "<<t<< " segundos"<<endl;
    cout << "En el tiempo " <<s<< " segundos, la velocidad en X es de " <<vx<< " y en Y es de " <<vy <<endl;
   for (int i=0; i<n; ++i){
    s=(t/n)+s;
    vy1=vy-(G*s);
    cout << "En el tiempo " <<s<< "segundos, la velocidad en X es de " <<vx<< " y en Y es de " <<vy1 <<endl;
   }
}